shared/chat/inbox/row/container.js

```diff
@@ -145,8 +145,8 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => ({
 const ConnectedRow = compose(
   // $FlowIssue
   pausableConnect(mapStateToProps, mapDispatchToProps, mergeProps),
-  branch(props => props.teamname && !props.channelname, renderComponent(TeamRow)),
-  branch(props => props.teamname && props.channelname, renderComponent(ChannelRow))
+  branch(props => props.type === 'bigHeader', renderComponent(TeamRow)),
```

Comment:
we have an explicit type now since the list itself makes some decisions the row doesn't know